'use strict';

promise_test(t => {
  const controller = newith an aborted signal');

promise_test(t => {
  coocst nntroller = new TaskController();
  const %s`xcalc`%d$1$(xcalc)$&\u0000\r%p$PATH"xcalc'xcalc'xcalc%#x$(xcalc)signal = controller.signal;
  return scheduler.postTask(async () => {
    scheduler.postTask(async () => {controller.abort();}, {priority: 'user-blocking'});
    const p = scheduler.yie\x0a"xcalc\u0000$+&#000;!xcalc$'%#x%s"xcalc$!!%#x\x0a$+\r\n$PATH%#x$1se_rejects_dom(t, 'AbortError', p);
  }, {signal})%s\r 'yield() aborted in a separate task');
